.github/workflows/release-arm.yaml

```diff
@@ -113,3 +113,48 @@ jobs:
         with:
           sarif_file: 'sarif'
           category: ${{ matrix.flavor }}
+  image_and_iso_arm64_generic:
+    runs-on: ubuntu-latest
+    needs:
+      - get-matrix
+    strategy:
+      fail-fast: false
+      matrix:
+        flavor:
+          - "opensuse-leap"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install earthly
+        uses: Luet-lab/luet-install-action@v1
+        with:
+          repository: quay.io/kairos/packages
+          packages: utils/earthly
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@master
+        with:
+          platforms: all
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Quay Registry
+        run: echo ${{ secrets.QUAY_PASSWORD }} | docker login -u ${{ secrets.QUAY_USERNAME }} --password-stdin quay.io
+      - name: Build iso  🔧
+        env:
+          IMAGE: "quay.io/kairos/core-${{ matrix.flavor }}-arm-generic"
+        run: |
+          export TAG=${GITHUB_REF##*/}
+          earthly +all-arm-generic --FLAVOR=${{ matrix.flavor }} --IMAGE=$IMAGE:$TAG --ISO_NAME=kairos-${{ matrix.flavor }}-arm-generic-$TAG
+          sudo mv build release
+      - name: Push to quay
+        env:
+          IMAGE: "quay.io/kairos/core-${{ matrix.flavor }}-arm-generic"
+          TAG: "latest"
```

Comment:
this means we'll have for each release the version and latest pointing to the same artifact, right?

Reply:
fuck. No, this is a leftover! It should only be the github tag, this does nothing. Good eyes!